projects/tags-input-demo/src/app/app.component.html

```diff
@@ -317,14 +316,33 @@
         (tagAdded)="onTagAdded($event)"
         (valueChanged)="onValChanged($event)"
         [config]="simpleTagsInputConfig"
+        [dropDownTemplate]="ddNestedTemplate"
         required="true"
         >
       </ti-angular-tags-input>
+      <ti-angular-tags-input formControlName="imageData" class="main__form__ti"
+        [tagsData]="sampleDataImage"
+        (tagAdded)="onTagAdded($event)"
+        (valueChanged)="onValChanged($event)"
+        [config]="imageTagsInputConfig"
+        [dropDownTemplate]="imageDropdownTemplate"
+        >
+      </ti-angular-tags-input>
       <div class="main__form__error" *ngIf="simpleForm.get('simpleData').dirty && simpleForm.get('simpleData').hasError('required')">
         Value for simple data is required
       </div>
     </form>
   </div>
+  <ng-template #imageDropdownTemplate let-items="items" let-config="config" >
+  <div class="users-list">
+    <tid-users-list-item
+      *ngFor="let data of items"
```

Comment:
`data` is a very generic word and should not be used in our programs in this way.
a better naming would be `let item of items`